### ISSUE TITLE
Is there an update available that can be compiled explicitly with the new bug fixes? Version is now at 1.2.0 but doesn't contain the latest commits.

### ISSUE THREAD

**Issue by Alexia8** [DavidPacioianu/InkPageIndicator#28]

**Comment by DavidPacioianu** [comment]
I will update it this weekend.

**Comment by Andrew-Quebe** [comment]
@DavidPacioianu see #31. I've already done this for you.

@Alexia8 If you want a new release now, you can add the following to your app's build.gradle:

```
repositories {
    // You have to point to my maven repo for 1.3.0
    maven { url "https://dl.bintray.com/vexigon/maven" }
}

dependencies {
    ...
    implementation 'com.pacioianu.david:ink-page-indicator:1.3.0'
    ...
}
```

**Comment by DavidPacioianu** [comment]
@Alexia8 I have just released version 1.3.0 of the library which includes your fixes and updated dependencies. Thank you for your contribution.
